src/queries/GetSystemProfileQuery.ts

```diff
@@ -61,6 +61,17 @@ export default gql`
       systemHasAPIGateway
       usesAiTech
     }
+    cedarContractsBySystem(cedarSystemId: $cedarSystemId) {
+      id: systemID
```

Comment:
Is there a reason to alias this here? Something to do with caching? (If so, might just be worth a quick inline comment explaining it, since it's not immediately clear looking at the code!)

Reply:
Yep for easy caching/consistency.  I like the note format suggestion!

Reply:
I don't think comments are supported in inline gql actually